docs/tutorials/HD143006_Part_2.py

```diff
@@ -0,0 +1,468 @@
+
+# +
+MODEL_PATH = "model.pt"
+
+new_config = (
+    {  # config includes the hyperparameters used in the function and in the optimizer
+        "lr": 0.3,
+        "lambda_sparsity": 7.076022085822013e-05,
+        "lambda_TV": 0.00,
+        "entropy": 1e-03,
+        "prior_intensity": 1.597766235483388e-07,
+        "epochs": 1000,
+    }
+)
+# -
+
+
+# We are now ready to run our cross validation loop, after it is done we will be reviewing our results in Tensorboard. We run this a few times while changing hyperparameters in the config to lower the cross validation score then compare all three with tensorboard.
+
+# +
+# %%time
+
+# new directory to write the progress of our first cross val. loop to
+cv_log_dir1 = logs_base_dir + "cv/cv1/"
+cv_writer1 = SummaryWriter(cv_log_dir1)
+os.makedirs(cv_log_dir1, exist_ok=True)
+
+cv_score1 = cross_validate(
+    model, new_config, k_fold_datasets, MODEL_PATH, writer=cv_writer1
+)
+print(f"Cross Validation Score: {cv_score1}")
+
+# +
+# %%time
+
+# new directory to write the progress of our second cross val. loop to
+cv_log_dir2 = logs_base_dir + "cv/cv2/"
+cv_writer2 = SummaryWriter(cv_log_dir2)
+os.makedirs(cv_log_dir2, exist_ok=True)
+
+new_config = (
+    {  # config includes the hyperparameters used in the function and in the optimizer
+        "lr": 0.3,
+        "lambda_sparsity": 1.0e-4,
+        "lambda_TV": 1.0e-4,
+        "entropy": 1e-02,
+        "prior_intensity": 2.0e-09,
+        "epochs": 850,
+    }
+)
+cv_score2 = cross_validate(
+    model, new_config, k_fold_datasets, MODEL_PATH, writer=cv_writer2
+)
+print(f"Cross Validation Score: {cv_score2}")
+
+
+# +
+# %%time
+
+# new directory to write the progress of our third cross val. loop to
+cv_log_dir3 = logs_base_dir + "cv/cv3/"
+cv_writer3 = SummaryWriter(cv_log_dir3)
+os.makedirs(cv_log_dir3, exist_ok=True)
+
+new_config = (
+    {  # config includes the hyperparameters used in the function and in the optimizer
+        "lr": 0.3,
+        "lambda_sparsity": 1.0e-3,
+        "lambda_TV": 1.2e-4,
+        "entropy": 1e-02,
+        "prior_intensity": 2.0e-09,
+        "epochs": 400,
+    }
+)
+
+cv_score3 = cross_validate(
+    model, new_config, k_fold_datasets, MODEL_PATH, writer=cv_writer3
+)
+print(f"Cross Validation Score: {cv_score3}")
+# -
+
+# And here are the results in the Tensorboard. As we run through this optimizer using different hyperparameters in the config file we can analyze the different results to work towards a lower cross validation score. Also note that while it may seem strange that the loss values are not lowest with the lowest cross validation scores, different config files increase the weight of some loss functions (and others remove some completely) so the loss values are not perfectly equal representations across each configuration and run.
+
+cv_log_dir = logs_base_dir + "cv/"
+# #%tensorboard --logdir {cv_log_dir}
+## uncomment the above line when running to view Tensorboard
```

Comment:
Would it be possible to show the trained model for the best CV score in the notebook directly? Since this isn't possible with tensorboard, I think you would need to add some more code to display the final image like in the other tutorials.